ttnn/cpp/ttnn/operations/eltwise/binary/binary.hpp

```diff
@@ -145,8 +145,14 @@ struct ExecuteBinary {
         const std::optional<ttnn::MemoryConfig> &memory_config = std::nullopt,
         const std::optional<Tensor> &optional_output_tensor = std::nullopt,
         std::optional<FusedActivations> activations = std::nullopt) {
-
-        return ExecuteBinary::execute_on_worker_thread(DefaultQueueId, input_tensor_a, scalar, dtype, operation::DEFAULT_OUTPUT_MEMORY_CONFIG, optional_output_tensor, activations);
+        return Binary::execute_on_worker_thread(
+            DefaultQueueId,
+            input_tensor_a,
+            scalar,
+            dtype,
+            operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
+            optional_output_tensor,
+            activations);
```

Comment:
Did you want to do this on this PR so that the optional_output_tensor is properly handled?
```
// TODO: add support for optional_output_tensors
            // auto optional_output_tensors = extract_args_to_vector(std::forward<args_t>(args)...,
            // std::optional<ttnn::Tensor>);
            ```

Reply:
will do in next PR